src/js/gpii-pouchdb-lucene.js

```diff
@@ -0,0 +1,140 @@
+    var iniContent = gpii.pouch.lucene.generateIniContent(that);
+    fs.writeFileSync(confFile, iniContent);
+
+    // Start the service using either the batch file on windows, or the shell script on anything else.
+    fluid.log("Starting couchdb-lucene...");
+    var script = os.platform().indexOf("win") === 0 ? "bin/run.bat" : "sh bin/run"; // The unix script is not always executable when it's unpacked.
+    that.process = child_process.exec(script, { cwd: workingDir});
+
+    that.process.stdout.on("data", that.waitForStartup);
+};
+
+// Ensure that the service is stopped on component destruction
+gpii.pouch.lucene.stopProcess = function (that) {
+    if (that.process) {
+        that.process.kill();
+    }
+};
+
+// Monitor the process' logs for "Accepting connections" and fire an `onStarted` event when the service is ready to
+// accept connections.
+gpii.pouch.lucene.waitForStartup = function (that, chunk) {
+    if (typeof chunk === "string" && chunk.indexOf("Accepting connections") !== -1) {
+        // pouchdb-lucene is actually ready a few milliseconds after the log message we look for.
+        setTimeout(function(){ that.events.onStarted.fire(that); }, that.options.startupDelay);
+    }
+    fluid.log(chunk);
+};
+
+gpii.pouch.lucene.generateIniContent = function(that) {
+    var output = "";
+    fluid.each(that.options.iniSettings, function(settings, section){
+        output += "\n[" + section + "]\n";
+
+        fluid.each(settings, function (value, key){
+            output += key + "=" + value + "\n";
+        });
+    });
+
+    return output;
+};
+
+fluid.defaults("gpii.pouch.lucene", {
+    gradeNames:   ["fluid.component"],
+    port:         9999,
+    startupDelay: 250, // How long to wait before reporting that couchdb-lucene is ready.
+    dbUrl:        "http://localhost:5986/ul",
+    // The settings we will write to couchdb-lucene's configuration file.  Each top level key will become a section
+    // entry, as in [lucene].  Each sub key-value pair will become an entry, as in dir=indexes
+    iniSettings: {
+        lucene: {
+            dir:                  "indexes",
+            host:                 "localhost",
+            port:                 "{that}.options.port",
+            limit:                25,
+            allowLeadingWildcard: false
+        },
+        local: {
+            url:  "{that}.options.dbUrl"
+        }
+    },
+    // The location of our distribution zip and working directory.  You should not need to change these.
+    zipPath:     zipPath,
+    zipFilename: zipFilename,
+    tmpDir:      tmpDir,
+    members: {
+        process: false
+    },
+    events: {
+        onStarted: null
+    },
+    listeners: {
+        "onCreate.init": {
+            funcName: "gpii.pouch.lucene.init",
+            args:     ["{that}"]
+        },
+        // TODO:  Review with Antranig and find a way to handle this bit of shutdown cleanly.
+        "onDestroy.stopProcess": {
```

Comment:
This should work without problems - I can't run the npm installer on Windows because of the following failure:

> rm -rf $npm_package_config_srcDir && git clone $npm_package_config_url $npm_package_config_srcDir

fatal: repository '$npm_package_config_url' does not exist

Can you tell me what you observe during this stopProcess execution?